vortex-file/src/strategy.rs

```diff
@@ -19,14 +20,61 @@ use vortex_layout::{LayoutStrategy, LocalExecutor, TaskExecutor};
 const ONE_MEG: u64 = 1 << 20;
 const ROW_BLOCK_SIZE: usize = 8192;
 
+enum ZoneMapConfiguration {
```

Comment:
I would actually stitch this in vortex-python. At rust level you can combine the strategy yourself?

Reply:
Hm, I looked at the dependencies of vortex-file and I thought we should minimize them but then I see that we include all the encodings